### PR TITLE
fix: don't create empty .libs directory

### PR DESCRIPTION
**Pull request by trim21**
close https://github.com/pypa/auditwheel/issues/488

only create `${package}.libs` when needed

### CODE REVIEW

**Comment by mayeut** [comment]
Thanks for your contribution, would you be able to add a test case for this as well (in `tests/integration/test_manylinux.py:AnyLinux`, the `testsimple` project can be used for the new test as it has no external dependency) ?

**Comment by trim21** [comment]
> Thanks for your contribution, would you be able to add a test case for this as well (in `tests/integration/test_manylinux.py:AnyLinux`, the `testsimple` project can be used for the new test as it has no external dependency) ?

where is `AnyLinux`? didn't find this with word search

File: tests/integration/test_manylinux.py [hunk 1]
```diff
@@ -1000,8 +999,14 @@ def test_build_wheel_compat(
             repaired_tag = f"{expect_tag}.{target_tag}"
         repaired_wheel = f"testsimple-0.0.1-{PYTHON_ABI}-{repaired_tag}.whl"
         assert repaired_wheel in filenames
+
         assert_show_output(manylinux_ctr, repaired_wheel, expect, True)
 
+        with zipfile.ZipFile(os.path.join(io_folder, repaired_wheel)) as z:
+            for file in z.namelist():
+                if file.startswith("testsimple.libs"):
+                    raise Exception("should not have empty .libs folder")
```
Comment (mayeut) [inline tests/integration/test_manylinux.py:1008]:
```suggestion
                assert not file.startswith("testsimple.libs"), "should not have empty .libs folder"
```

Comment (trim21) [reply tests/integration/test_manylinux.py]: thanks, forget there is an assert statment...

**Review by auvipy — APPROVED** [review]

**Pull request merged**